src/main/java/website/frontrow/Launcher.java

```diff
@@ -181,6 +174,72 @@ private void startScheduler(Game game)
                 }
             });
         }
+        else
+        {
+        	map.put(KeyEvent.VK_A, () ->
+            {
+                addToLog("[KEY]\t< \'<-\' > Pressed.");
+                game.getPlayers().get(0).goLeft();
+            });
+
+            map.put(KeyEvent.VK_D, () ->
+            {
+                addToLog("[KEY]\t< \'->\' > Pressed.");
+                game.getPlayers().get(0).goRight();
+            });
+
+            map.put(KeyEvent.VK_W, () ->
+            {
+                addToLog("[KEY]\t< \' \' > Pressed.");
+                game.getPlayers().get(0).jump();
+            });
+
+            map.put(KeyEvent.VK_SPACE, () ->
+            {
+                addToLog("[KEY]\t< \'Z\' > Pressed.");
+
+                if(game.isRunning())
+                {
+                    Player p = game.getPlayers().get(0);
+                    game.getLevel().addUnit(
+                            new Bubble(p.getLocation(),
+                                    new Point(p.getDirection().getDeltaX() * 4, 0),
```

Comment:
It's more that it might prove useful to have this as a constant as it's used in 2 parts of the code, so when it's constant we might have an easier time adjusting it if we might feel the need to do so.